internal/k8s/controller.go

```diff
@@ -763,7 +763,7 @@ func (c *AviController) SetupEventHandlers(k8sinfo K8sinformers) {
 					utils.AviLog.Warnf("calico blockaffinity spec not found: %+v", err)
 					return
 				}
-				key := utils.NodeObj + "/" + specJSON["name"]
+				key := utils.NodeObj + "/" + specJSON["node"]
```

Comment:
Are we planning to add any UTs to test this?

Reply:
I am merging this PR. @DixitAakash: Lets have UT/FT to test this functionality